README.md

```diff
@@ -1545,6 +1545,7 @@ A [pastebin](https://en.wikipedia.org/wiki/Pastebin) is a type of online content
 - [Sup3rS3cretMes5age](https://github.com/algolia/sup3rS3cretMes5age) - Very simple (to deploy and to use) secret message service using Hashicorp Vault as a secrets storage. `MIT` `Go`
 - [wantguns/bin](https://github.com/wantguns/bin) - Minimal pastebin for both textual and binary files shipped in a single statically linked binary. ([Demo](https://basedbin.fly.dev)) `GPL-3.0` `Rust`
 - [Wastebin](https://github.com/matze/wastebin) - Lightweight, minimal and fast pastebin with an SQLite backend. ([Demo](https://bin.bloerg.net)) `MIT` `Rust`
+- [YABin](https://github.com/Yureien/YABin) - A pastebin that aims to have ALL the good features while being simple. Supports optional E2E encryption, CLI app, syntax highlighting, minimalistic UI, APIs, keyboard shortcuts, and more. It can even be run in serverless environments. ([Demo](https://bin.sohamsen.me/)) `MIT` `Typescript`
```

Comment:
Couple changes:

- Please change `ALL` to `All` if youre going to have it but I would say that `A pasetbin with plentiful features while remaining simple`.
- Please change `Typescript` to `Nodejs` as Typescript is not a serverside langauage

Reply:
Hey @KieranRobson, I have implemented the changes. One note: It requires only NodeJS on the server side. The CLI is not used on the server side, it's only for client side. Kept it in a monorepo format, the Docker image does not contain the CLI.

Thanks for reviewing!

Reply:
`Nodejs/Docker` would be acceptable since both installation methods are available. But I approve it in the current form.